tree/ntuple/v7/test/ntuple_packing.cxx

```diff
@@ -7,18 +7,20 @@
 #include <type_traits>
 #include <utility>
 
-template <typename PodT, typename NarrowT, ROOT::Experimental::EColumnType ColumnT>
+#include "ROOT/RColumnElement.hxx"
```

Comment:
This include should probably go to `ntuple_test.hxx` if the `using declarations are there...